### ISSUE TITLE
if geolocation api fails, cant creat button

### ISSUE THREAD

**Issue by albjeremias** [helpbuttons/helpbuttons#844]
when creating a button, if the geolocation api fails, it will fail to get the address, so the button can't be created without an address

**Comment by albjeremias** [comment]
dont known a solution yet, maybe the user can always edit the address? and it adds a a list of places? and the admin can mark places to be shown on the map! so when you add a place, you need a name and an address?

**Comment by albjeremias** [comment]
a solution should be that the user can add the address and the gps position manually

**Comment by watchoutfreedom** [comment]
> a solution should be that the user can add the address and the gps position manually

yes, and maybe in the future admins can review the address to be added to the database?

**Comment by albjeremias** [comment]
yes, ill make something like in gancio.
